pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -112,7 +112,7 @@ func (s *Service) Handle(msg *service.DIDCommMsg) error {
 
 	logger.Infof("entered into Handle exchange message : %s", msg.Payload)
 
-	if !msg.Outbound && aEvent == nil {
```

Comment:
was the msg.Outbound implied?

Reply:
Outbound flag is used in state machine. The inbound dispatcher never sets this (always false) https://github.com/hyperledger/aries-framework-go/pull/482#discussion_r334157477.

Reply:
Hmmm. I think the way this code is being refactored implies that the same type shouldn’t be used... commented above.